site/cypress/e2e/gradeables/gradeable.spec.js

```diff
@@ -0,0 +1,41 @@
+describe('Tests cases revolving around gradeable access and submition', () => {
+    ['student','instructor'].forEach((user) => {
```

Comment:
Why not also check for other access levels as well?

Reply:
I was thinking that student and instructor is good enough regular tests because the ta access level has almost the same access level as a instructor and if the instructor doesn't give access to the ta, then the gradeable wouldn't even show, but I realized that we might add more restrictions in the future, so I will add a ta too.

Reply:
if you're going to add a full access grader, might as well add a limited access grader (users `ta` and `grader` for example)